src/frontend/Theme.ts

```diff
@@ -26,7 +26,6 @@ export const GlobalStyle = createGlobalStyle`
 
   body {
     margin: 0;
-    box-sizing: content-box;
```

Comment:
Denne globale stilen gjorde at nye komponenter fra ds-react ble alt for store 🤔 Har klikket meg gjennom løsningen og ser ikke ut som at det skapte noen problemer når jeg fjernet den.